native_client/deepspeech.cc

```diff
@@ -354,6 +354,27 @@ DS_AddHotWord(ModelState* aCtx,
   return DS_ERR_SCORER_NOT_ENABLED;
 }
 
+int
+DS_EraseHotWord(ModelState* aCtx,
+                const char* word)
+{
+  if (aCtx->scorer_) {
+    aCtx->hot_words_.erase(word);
+    return DS_ERR_OK;
```

Comment:
shouldn't that be checked more thoroughly?

Reply:
what do you mean?

Reply:
yeah, but what happens if `erase` fails?

Reply:
addressed in c90b05478fed932c88a9254f96700c9be9136c19